src/test/java/io/qdrant/client/ApiKeyTest.java

```diff
@@ -61,7 +61,8 @@ public void client_without_api_key_cannot_connect() {
 			Throwable cause = executionException.getCause();
 			assertEquals(StatusRuntimeException.class, cause.getClass());
 			StatusRuntimeException statusRuntimeException = (StatusRuntimeException) cause;
-			assertEquals(Status.Code.PERMISSION_DENIED, statusRuntimeException.getStatus().getCode());
+			// FIXME: This assertion fails because the server returns UNAUTHENTICATED instead of PERMISSION_DENIED
+			// assertEquals(Status.Code.PERMISSION_DENIED, statusRuntimeException.getStatus().getCode());
```

Comment:
Recommend annotating method with `@Disabled("<reason>")` as it's more discoverable

Reply:
👍👍

Will switch the check to `Status.Code.UNAUTHENTICATED` once the `1.9` image is out.

Reply:
@Anush008 could you arrange this one? The image should be available now.

Reply:
Will do.